src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/AgreementNotSigned.cshtml

```diff
@@ -22,7 +22,13 @@
             <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
             <div class="govuk-warning-text__text">
                 <span class="govuk-warning-text__assistive">Warning</span>
-                <p>@Model.LegalEntityName does not have an accepted agreement with the Department for Education (<abbr>DfE</abbr>). Without an accepted agreement you can add apprentice details, but will not be able to authorise payments to training providers.</p>
+                @if (Model.CanContinueAnyway)
```

Comment:
Looks like someone half attempted this previously putting the text into a variable above lines 11-13.

Could we delete or use this variable please?

Reply:
I didn't see that, will do that now.